package.json

```diff
@@ -87,11 +88,18 @@
     "slick-carousel": "1.5.8",
     "source-map-support": "0.3.2",
     "style-loader": "0.12.3",
-    "tap": "2.0.0",
+    "tap": "7.1.2",
```

Comment:
note to reviewer: tap needed to be upgraded for the latest coverage features. I bumped it up to the latest.